tests/forms_tests/widget_tests/test_widget.py

```diff
@@ -24,3 +26,33 @@ def test_no_trailing_newline_in_attrs(self):
     def test_attr_false_not_rendered(self):
         html = '<input type="None" name="name" value="value" />'
         self.check_html(Input(), 'name', 'value', html=html, attrs={'readonly': False})
+
+    def test_custom_widget_renderer_warning(self):
```

Comment:
Put this in a separate `test_render_deprecation.py` file so we can just delete it at the end of the deprecation. It reduces the chance of future backport conflicts also.